backend/src/main/java/com/woowacourse/kkogkkog/infrastructure/event/PushAlarmPublisher.java

```diff
@@ -0,0 +1,29 @@
+package com.woowacourse.kkogkkog.infrastructure.event;
+
+import com.woowacourse.kkogkkog.member.domain.Member;
+import com.woowacourse.kkogkkog.member.domain.MemberHistory;
+import com.woowacourse.kkogkkog.member.domain.Workspace;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PushAlarmPublisher {
```

Comment:
MemberHistory라는 JPA 엔티티를 사용넘겨받는데, 혹시 모르니 `@Transactional`도 붙일까요?

```suggestion
@Component
@Transactional
public class PushAlarmPublisher {
```